### PR TITLE
PW-DSS-PEPPER-1003 -Enrollment

### PR DESCRIPTION
**Pull request by GiCharkviani**
[PEPPER-1003](https://broadworkbench.atlassian.net/jira/software/projects/pepper/boards/164?assignee=621677042d057500723f90e4&selectedIssue=PEPPER-1003)

**Note: Tests don't include screenshot comparison** 

[video.webm](https://github.com/broadinstitute/ddp-angular/assets/77500504/9967aa8b-a4c2-429f-9970-4925d80d768d)

[PEPPER-1003]: https://broadworkbench.atlassian.net/browse/PEPPER-1003?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: playwright-e2e/config/.env.staging [hunk 1]
```diff
@@ -57,3 +57,8 @@ BRAIN_USER_PASSWORD=
 LMS_USER_EMAIL=
 LMS_USER_PASSWORD=
 LMS_BASE_URL=https://lms.staging.datadonationplatform.org
+
+# MBC
+MBC_BASE_URL=https://mbc.dev.datadonationplatform.org
```
Comment (aweng98) [inline playwright-e2e/config/.env.staging]: should be `mbc.staging`

File: playwright-e2e/config/.env.test [hunk 2]
```diff
@@ -57,3 +57,8 @@ BRAIN_USER_PASSWORD=
 LMS_USER_EMAIL=
 LMS_USER_PASSWORD=
 LMS_BASE_URL=https://lms.test.datadonationplatform.org
+
+# MBC
+MBC_BASE_URL=https://mbc.dev.datadonationplatform.org
```
Comment (aweng98) [inline playwright-e2e/config/.env.test]: should be `mbc.test`.

File: playwright-e2e/dss/pages/mbc/mbc-follow-up-survey-1.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
+import {MBCPageBase} from './mbc-page-base';
+import {expect, Locator, Page} from '@playwright/test';
+import {waitForNoSpinner} from '../../../utils/test-utils';
```
Comment (aweng98) [inline playwright-e2e/dss/pages/mbc/mbc-follow-up-survey-1.ts]: Relative imports are discourged.

**Review by aweng98 — COMMENTED** [review]
Looks Good! Need to update `export-playwright-en.sh` for MBC variables. 
https://github.com/broadinstitute/ddp-angular/blob/develop/.circleci/export-playwright-env.sh

File: playwright-e2e/tests/mbc/mbc-enrollment.spec.ts [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+import {MBCJoinPage} from '../../dss/pages/mbc/mbc-join-page';
+import * as auth from '../../authentication/auth-lms';
+import {logParticipantCreated} from '../../utils/log-utils';
+import {MBCSurveyAboutPage} from '../../dss/pages/mbc/mbc-survey-about-page';
+import {MBCPatientsData} from '../../dss/pages/mbc/mbc-patient-type';
+import {MBCResearchConsentPage} from '../../dss/pages/mbc/mbc-research-consent-page';
+import {MBCMedicalReleasePage} from '../../dss/pages/mbc/mbc-medical-release-page';
+import {MBCFollowUpSurvey1} from '../../dss/pages/mbc/mbc-follow-up-survey-1';
+
+const {MBC_USER_EMAIL, MBC_USER_PASSWORD, MBC_BASE_URL, SITE_PASSWORD} = process.env;
```
Comment (aweng98) [inline playwright-e2e/tests/mbc/mbc-enrollment.spec.ts:15]: Do you also want to create a new `fixture` for this study?

File: playwright-e2e/config/.env.staging (hunk 1 above)
Comment (GiCharkviani) [reply playwright-e2e/config/.env.staging]: Oh, forgot. Thanks.
Done!

File: playwright-e2e/config/.env.test (hunk 2 above)
Comment (GiCharkviani) [reply playwright-e2e/config/.env.test]: Done!

File: playwright-e2e/tests/mbc/mbc-enrollment.spec.ts (hunk 4 above)
Comment (GiCharkviani) [reply playwright-e2e/tests/mbc/mbc-enrollment.spec.ts:15]: In the future would be good.

File: playwright-e2e/dss/pages/mbc/mbc-follow-up-survey-1.ts (hunk 3 above)
Comment (GiCharkviani) [reply playwright-e2e/dss/pages/mbc/mbc-follow-up-survey-1.ts]: missed that :D Fixed!

**Comment by aweng98** [comment]
Hi, I don't find changes to `update export-playwright-en.sh`.

**Review by aweng98 — APPROVED** [review]

**Pull request merged**